client/components/PageList/PageList.jsx

```diff
@@ -67,7 +80,6 @@ class Pager extends React.Component {
     }
```

Comment:
i+=1 ---> i++ is good shorthand when incrementing.  line 78

Reply:
I know but eslint hates it 😄